python/navigation/src/algos.cc

```diff
@@ -395,7 +395,6 @@ addAlgorithmClasses(py::module &m)
                         return PyAlgoWrapper<InfoMaxType>{
                             state[0].cast<cv::Size>(),
                             state[1].cast<float>(),
-                            state[2].cast<float>(),
```

Comment:
What's this fix for?

Reply:
The algos fix is just a place I forgot to remove the tanh scaling from before. Editable mode is maybe not super helpful in this context as there's not much/no python code but I generally use it for packages I'm developing/have checked out of git and want to version that way - Like ``python setup.py develop`` before it, it just sticks a link into your python site-packages folder rather than making a copy.

Reply:
Theoretically Jenkins is still running on BoB robotics - it doesn't look to be producing wheels though! I haven't dug deeper for a while but replacing Jenkins with github actions and local runners is on the near-future todo list after I finish the current round of GeNN development.

Reply:
I'm happy to have a go -- we've got similar workflows to copy from. For non-CUDA-dependent bits, we could just run them on the default runners. They'll be free because BoB robotics is open source.